compose/docker-compose.acceptance-tests.yml

```diff
@@ -17,22 +17,27 @@ services:
       GRID_DEBUG: "false"
 
   selenium-firefox:
-    image: "selenium/node-firefox:3.7.1-beryllium"
+    image: "selenium/node-firefox-debug:3.7.1-beryllium"
     links:
       - "selenium-hub"
       - "nginx"
     shm_size: 2g  # See https://goo.gl/dkZzHh
+    ports:
+      - "62102:5900"
     environment:
+      FIREFOX_VERSION: "47.0.1"
```

Comment:
Is this here for a reason? I noticed that https://hub.docker.com/r/selenium/node-firefox/~/dockerfile/ takes a `FIREFOX_VERSION` but it's a build argument not an environment variable. Is this being used somewhere else?